### PR TITLE
[4657] Reactify Global App (About)

### PR DESCRIPTION
**Pull request by aachavarria**
https://github.com/craftercms/craftercms/issues/4657

### CODE REVIEW

File: static-assets/scripts/main.js [hunk 1]
```diff
@@ -1099,6 +1099,19 @@
     }
   ]);
 
+  app.controller('AboutCtrl', [
```
Comment (rart) [inline static-assets/scripts/main.js:916]: Make sure to clean up any requests on this side that will now be done on the react side

File: ui/app/src/components/About/About.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import GlobalAppToolbar from '../GlobalAppToolbar';
+import { FormattedMessage } from 'react-intl';
+import { useEnv, useSelection } from '../../utils/hooks';
+import Paper from '@material-ui/core/Paper';
+import useStyles from './style';
+import CrafterCMSLogo from '../Icons/CrafterCMSLogo';
+import Box from '@material-ui/core/Box';
+import { Link, Typography } from '@material-ui/core';
+
+export default function About() {
```
Comment (rart) [inline ui/app/src/components/About/About.tsx]: AboutCrafterCMSView

File: ui/app/src/components/About/About.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+import React from 'react';
+import GlobalAppToolbar from '../GlobalAppToolbar';
+import { FormattedMessage } from 'react-intl';
+import { useEnv, useSelection } from '../../utils/hooks';
+import Paper from '@material-ui/core/Paper';
+import useStyles from './style';
+import CrafterCMSLogo from '../Icons/CrafterCMSLogo';
+import Box from '@material-ui/core/Box';
+import { Link, Typography } from '@material-ui/core';
```
Comment (rart) [inline ui/app/src/components/About/About.tsx]: Fix

File: ui/app/src/components/About/style.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/*
```
Comment (rart) [inline ui/app/src/components/About/style.ts]: style.ts => styles.ts

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/scripts/main.js (hunk 1 above)
Comment (aachavarria) [reply static-assets/scripts/main.js:916]: That requests will be cleaned on the next reactify(account) because they were using the same controller

**Review by rart — APPROVED** [review]

**Comment by sumerjabri** [comment]
@aachavarria please fix conflict

**Comment by rart** [comment]
@sumerjabri fixed

**Pull request merged**